.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 
 orbs:
   owasp: entur/owasp@0.0.10
-  hmpps: ministryofjustice/hmpps@1.1.2
+  hmpps: ministryofjustice/hmpps@2.1.0
```

Comment:
The original orb made a reference to a repo link which has subsequently changed.  The latest orb fixes this.